src/utils/env.ts

```diff
@@ -68,9 +68,6 @@ export const env = createEnv({
       ),
     PORT: z.coerce.number().default(3005),
     HOST: z.string().default("0.0.0.0"),
-    ACCESS_CONTROL_ALLOW_ORIGIN: z
```

Comment:
Btw since we removed this here, we should add it to the defaults in the `getConfiguration` functions

Reply:
Added [here](https://github.com/thirdweb-dev/engine/pull/372/commits/a71db82bd2b9eaf5b991fa4e9afedb337e935e85)